src/pages/fundamentals/typography.hbs

```diff
@@ -93,26 +101,27 @@ styles:
 @include text-body-md;
 @include text-body-sm;
 @include text-body-xs;
-@include text-cta;</code></pre>
+@include text-body-cta;</code></pre>
 </div>
 
 <p>Those mixins include media queries to adjust the sizes at key breakpoints. Text gets bigger as the viewport gets bigger. You can resize your window to see this in action on the sample above or on the <a href="{{@root.baseurl}}/demos/type-scale.html">demo page</a>.</p>
 
-<p>For example, here’s the 4XL mixin:</p>
```

Comment:
Sigh... how long has this been here... it's an artifact from the very beginnings of Protocol. At an early stage we didn't split the type scale by "display" and "body" and just used sizes from 4XL down to I think 3XS, but quickly abandoned that and I clearly failed to ever update this reference. We also didn't used repeated Xes in that scale because "xxxxl" is harder to parse. Ah memories...